Library/Formula/red5.rb

```diff
@@ -0,0 +1,48 @@
+class Red5 < Formula
+  desc "Open Source Flash Server written in Java"
+  homepage "https://github.com/Red5/red5-server"
+  url "https://github.com/Red5/red5-server/releases/download/v1.0.6-RELEASE/red5-server-1.0.6-RELEASE-server.tar.gz"
+  version "1.0.6"
+  sha256 "4aee3f312d16e09495e5010cf95bc7b8ed48620163950425a1f147f17aaab5f7"
+
+  depends_on :java => "1.7+"
+
+  def install
+    rm Dir["*.bat"]
+
+    (bin/"red5").write_env_script libexec/"red5.sh", Language::Java.java_home_env("1.7+")
+
+    # Move config files into etc
+    (etc/"red5").install Dir["conf/*"]
+    libexec.install_symlink "#{etc}/red5" => "conf"
+    rm_rf "conf"
+
+    # Move webapps into var
+    (var/"red5/webapps").install Dir["webapps/*"]
+    libexec.install_symlink "#{var}/red5/webapps" => "webapps"
+    rm_rf "webapps"
+
+    # Install others files
+    libexec.install Dir["*"]
+
+    inreplace "#{libexec}/red5.sh" do |s|
+      # Configure RED5_HOME
+      s.sub!("export RED5_HOME=`pwd`", "export RED5_HOME=#{libexec}")
+      s.sub!("# start Red5", "# start Red5\ncd `dirname $0`")
+    end
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!

Alternatively, consider writing a wrapper script instead of rewriting this.